app/models/organization_wizzard.rb

```diff
@@ -0,0 +1,31 @@
+class OrganizationWizzard
+  include Rails.application.routes.url_helpers
+
+  def initialize(organization)
+    @organization = organization
+  end
+
+  def ready?
+    have_account? && have_categories?
+  end
+
+  def not_ready?
```

Comment:
@dissident you may leave it as is if you will show me at least one method in ruby or rails which have `not_` in name and have opposite method without `not_` ;)